src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.constant.Color;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBERS_OF_FIGURES = 5;
+    private static final int DEFAULT_RADIUS_FOR_CIRCLE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int type = random.nextInt(NUMBERS_OF_FIGURES);
+        double areaValue = random.nextDouble();

```

Comment:
first of all - naming, this is not an area yet, it is just a value for one of the figure parameters,
second of all, with that approach every figure would always have the same parameters that are units measured, for example, a rectangle would have both sides the same, so it would now be also always a square

Also, we can encapsulate it a bit more and add a method like `getRandomDouble()` to call it instead of direct random call to avoid code duplication, that would also be a great example of the `random` re-usage in multiple methods